### PR TITLE
Replacing Kotlin Synthetics with View Binding

### PR DESCRIPTION
**Pull request by Sudhindra3000**

### CODE REVIEW

**Comment by ranaaditya** [comment]
> _No description provided._

Thanks @Sudhindra3000 I will review this PR ASAP !

File: app/src/main/java/org/pragyan/dalal18/ui/LoginActivity.kt [hunk 1]
```diff
@@ -40,6 +40,8 @@ import javax.inject.Inject
 
 class LoginActivity : AppCompatActivity() {
 
+    private lateinit var binding: ActivityLoginBinding
```
Comment (avikumar15) [inline app/src/main/java/org/pragyan/dalal18/ui/LoginActivity.kt:43]: Implement a Property Delegate to handle the lifecycle of Bindings. Checkout [fest-android](https://github.com/delta/fest-android/)

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Tue Feb 25 19:14:47 IST 2020
+#Mon Jan 04 20:46:13 IST 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip
```
Comment (avikumar15) [inline gradle/wrapper/gradle-wrapper.properties:6]: Revert this to the older version for now

Comment (Sudhindra3000) [reply gradle/wrapper/gradle-wrapper.properties:6]: Gradle Version 6.1.1 and Gradle Plugin Version 4.0.1 are required for ViewBinding

**Review by avikumar15 — APPROVED** [review]

**Review by ranaaditya — APPROVED** [review]

**Comment by ranaaditya** [comment]
Merged this PR into view-binding branch, soon will merge into dalal'21 after proper testing.
Thanks @Sudhindra3000 for your precious contribution :partying_face:

**Pull request merged**